redaxo/src/core/pages/packages.php

```diff
@@ -212,9 +221,41 @@
 
     $content .= '</tbody>
             </table>';
-
+			
+    $content .= '
+        <script type="text/javascript">
+        <!--
+        jQuery(function($) {
+            var table = $("#rex-js-table-available-packages-addons");
+            var tablebody = table.find("tbody");
+            
+            $("#rex-js-available-addon-search .form-control").keyup(function () {
+                table.find("tr").show();
+                var search = $(this).val().toLowerCase();
+                if (search) {
+                    table.find("tbody tr").each(function () {
+                        var tr = $(this);
+                        if (tr.text().toLowerCase().indexOf(search) < 0) {
+							tr.hide();
+                        }
+                    });
+                }
+                else
+                {
+                   // else
```

Comment:
der `else´-Block könnte weg.

Reply:
Ja, habe ich noch drin behalten, um später auf keine Treffer einzugehen.

Reply:
> Ja, habe ich noch drin behalten, um später auf keine Treffer einzugehen.

das klingt nach einem weiteren PR :wink:.

Ich würde den vor dem Mergen eben raus nehmen. Den kannst du dann in einem möglichen Anschluss-PR wieder anfügen.